src/main/java/redis/clients/jedis/JedisPool.java

```diff
@@ -206,14 +190,6 @@ public JedisPool(final GenericObjectPoolConfig poolConfig, final String host, in
         sslSocketFactory, sslParameters, hostnameVerifier);
   }
 
-  public JedisPool(final GenericObjectPoolConfig poolConfig, final String host, int port,
```

Comment:
@tgrall I tried to avoid creating too many new constructors. So I skipped adding user param where there exist only one timeout param and all ss params.